hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -100,42 +111,89 @@ public void run() {
 
   public void triggerContainerHealthCheck() {
     lock.writeLock().lock();
+    Map<String, Map<String, Long>> unhealthyContainerStateStatsMap;
```

Comment:
Add a comment above this line stating the type of keys and values in the map.

Reply:
Also consider using enum directly rather than converting it to `String` for the key `unhealthyState`.

Reply:
Done.